src/metabase/util/ssh.clj

```diff
@@ -6,12 +6,20 @@
 
 (def ^:private default-ssh-timeout 30000)
 
+(defn ssh-add-key
+  "Adds SSH private key to Jsch class."
+  [jsch key-file-name]
+  (log/debug "Adding ssh key:" key-file-name)
+  (when-not (string/blank? key-file-name) (doto ^com.jcraft.jsch.JSch jsch (.addIdentity key-file-name)))
```

Comment:
@camsaul I tested it out on my local build of Metabase at work, but we haven't deployed an actual server yet. I haven't written any formal tests either :(

While I was testing it out, I was using Metabase every day for about a week to connect to a remote MySQL server via the tunnel + ssh key.

Automated tests are on my to-do list, I just haven't had the time to actually write them. Any help would be greatly appreciated! :)